ytypes/node_exported_test.go

```diff
@@ -818,3 +849,230 @@ func TestSetNode(t *testing.T) {
 		}
 	}
 }
+
+func TestDeleteNode(t *testing.T) {
```

Comment:
does this need to be removed from node_test.go

Reply:
Updated names.

Reply:
These are just for ordered maps which need access to `ctestschema` but would cause a circular dependency if it's in the same package.